evalml/objectives/cost_benefit_matrix.py

```diff
@@ -35,12 +35,20 @@ def __init__(self, true_positive, true_negative, false_positive, false_negative)
         self.false_positive = false_positive
         self.false_negative = false_negative
 
-    def objective_function(self, y_true, y_predicted, X=None, sample_weight=None):
+    def objective_function(
+        self,
+        y_true,
+        y_predicted,
+        y_train=None,
+        X=None,
+        sample_weight=None,
+    ):
         """Calculates cost-benefit of the using the predicted and true values.
 
         Args:
             y_predicted (pd.Series): Predicted labels.
             y_true (pd.Series): True labels.
+            y_train (pd.Series): Training labels.
```

Comment:
For this and other metrics where `y_train` isn't used, can you mention that this is ignored?